doc/nrf/security/ap_protect.rst

```diff
@@ -170,43 +170,144 @@ Based on the available implementation types, you can configure the access port p
 
 .. _app_approtect_ncs_lock:
 
-Enabling AP-Protect with :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK`
-====================================================================
+Enabling software AP-Protect with :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK`
+=============================================================================
+
+Setting the :kconfig:option:`CONFIG_NRF_APPROTECT_LOCK` Kconfig option to ``y`` and compiling the firmware enables the software access protection mechanism for SoCs of the nRF53 Series and the SoC revisions of the nRF52 Series that feature the hardware and software type of AP-Protect.
+
+Enabling the Kconfig option writes the debugger register in the ``SystemInit()`` function to lock the access port protection at every boot.
+In addition to this, the UICR.APPROTECT register should be written as instructed in :ref:`app_approtect_uicr_approtect`.
```

Comment:
There are still 2 occurrences of `UICR.APPROTECT` that could be put between backticks.